src/TheGuardianIn2020Banner/index.tsx

```diff
@@ -50,6 +50,28 @@ export const TheGuardianIn2020Banner: React.FC<Props> = ({
 }: Props) => {
     const [showBanner, setShowBanner] = useState(true);
 
+    const onClick = (internalButtonId: number): void => {
+        setShowBanner(false);
```

Comment:
Do we want to be closing the banner here?

Reply:
Good question - my take on this was that we would want to close it, but I don't feel very strongly about it.

If a user were to middle click for example, they would then have to close the banner after following the link. Having both buttons close the banner makes it easier to split the audience in the next step if we were using button 0 clicks versus button 1 clicks as branched advancement criteria for a canvas (because they can't click on both)

On the other hand, a user probably wouldn't expect clicking that link to close the banner, so I wasn't certain this was the right behaviour myself. 

What do you reckon is best?

Reply:
I think from a user perspective I'd expect command clicking on the link to open in a new tab and leave the banner up. That's how RR banners behave, so having consistency here would be good.

Good point about splitting the audience in the canvas, and the fact that the user could take both paths. I think as long as we let marketing know what the behaviour is then they can design the canvases accordingly.